core/src/main/java/org/sonarsource/sonarlint/core/container/storage/StorageManager.java

```diff
@@ -128,19 +132,18 @@ public GlobalStorageStatus getGlobalStorageStatus() {
 
   @CheckForNull
   private GlobalStorageStatus initStorageStatus() {
-    Path updateStatusPath = getUpdateStatusPath();
-    if (Files.exists(updateStatusPath)) {
-      final Sonarlint.UpdateStatus updateStatusFromStorage = ProtobufUtil.readFile(updateStatusPath, Sonarlint.UpdateStatus.parser());
-      final boolean stale = (updateStatusFromStorage.getSonarlintCoreVersion() == null) ||
-        !updateStatusFromStorage.getSonarlintCoreVersion().equals(VersionUtils.getLibraryVersion());
+    Path storageStatusPath = getStorageStatusPath();
+    if (Files.exists(storageStatusPath)) {
+      final Sonarlint.StorageStatus storageStatus = ProtobufUtil.readFile(storageStatusPath, Sonarlint.StorageStatus.parser());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "storageStatus" which hides the field declared at line 55. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AHiddenFieldCheck)